packages/inputs-tiny/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@goodhood/inputs-tiny",
```

Comment:
would still opt for `@goodhood/inputs-minimal`, don't like `tiny` as a theme name because ... well ... are the inputs tiny or where does it come from? 

still open for better alternatives